src/CommandingOfficers/Commander.java

```diff
@@ -235,6 +235,20 @@ public UnitModel getUnitModel(UnitEnum unitType)
 
     return um;
   }
+  
+  /**
+   * Returns a character to be displayed on the unit while the map is seen from your perspective.
+   * Primary usage should be pieces of info that aren't otherwise immediately apparent from the map.
+   */
+  public char getSymbol(Unit unit)
```

Comment:
I think a different name would be more meaningful. Maybe `getUnitMarker/ing` or `getCustomSymbol` something. The current name sounds like maybe each Commander has an emblem.